cypress/e2e/models/migration/applicationinventory/analysis.ts

```diff
@@ -165,21 +165,16 @@ export class Analysis extends Application {
 
     public static selectLanguage(language: Languages) {
         cy.wait(2 * SEC);
-        click(`${actionSelectToggle} > button`);
-        clickByText("button", language);
+        click(actionSelectToggle);
+        clickByText("div", language);
+        click(actionSelectToggle);
     }
 
-    protected selectTarget(target: string[]): void {
-        for (let i = 0; i < target.length; i++) {
-            if (["OpenJDK 11", "OpenJDK 17", "OpenJDK 21"].includes(target[i])) {
-                click(openjdkToggleButton);
-                clickByText(dropDownMenu, target[i]);
-            } else if (["camel:3", "camel:4"].includes(target[i])) {
-                click(camelToggleButton);
-                clickByText(dropDownMenu, target[i]);
-            }
-            cy.get("div.pf-v5-c-empty-state__content").contains(target[i]).click();
-        }
+    protected selectTarget(targets: string[]): void {
+        targets.forEach((targetName) => {
+            let cardId = `#target-card-${targetName.replace(/\s+/g, "-")}`;
+            cy.get(cardId).find('input[type="checkbox"]').check({ force: true });
+        });
```

Comment:
@ibolton336 have you tried to execute this TC `OpenJDK21 Source + dependencies analysis on daytrader app` under `tackle-ui-tests/cypress/e2e/tests/migration/applicationinventory/analysis/source_analysis.test.ts` ?

I beliveve this test will fail with those targets that have a dropdown with multiple options, like openjdk.

I'd try it myself but we're having some issues deploying Tackle. If you want to wait I'll give it a try when the deployment issue is solved.

Reply:
Removed this bit - @abrugaro